lib/mongo/monitoring/event/command_failed.rb

```diff
@@ -109,13 +113,15 @@ def summary
         # @param [ BSON::Document ] failure The error document, if any.
         # @param [ Float ] duration The duration of the command in seconds.
         # @param [ Object ] service_id The service id, if any.
+        # @param [ false | true ] force_redaction Whether the error document
+        #   should be redacted.
         #
         # @return [ CommandFailed ] The event.
         #
         # @since 2.1.0
         # @api private
         def self.generate(address, operation_id, payload, message,
-          failure, duration, service_id: nil
+          failure, duration, service_id: nil, force_redaction: fasle
```

Comment:
```suggestion
          failure, duration, service_id: nil, force_redaction: false
```

Reply:
I am wondering how this could pass specs! Thanks.